### DISCUSSION TITLE
Font size

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#56] [Ideas]
We should take care of different font size usage.
Beginning with variables telling all programs what is the actual font width & height so we can replace the hard coded 10 value.

I have motivation for testing a anti-aliased 6x12 font with 4 transparency level stored in PNG. It allow 640/6=106 colon and 480/12=40 line in VGA!

I used the BleskOS calculator for this and it is not yet very efficient:
- focus not automatically in the first operand
- +,/,+,- key not working
- TAB not working to change operand focus
- right part of the screen could be wisely used to display historical data of the last graphic_screen_y / current_font_height operation :-)
-

**Comment by VendelinSlezak** [comment]
Well, whole thing with scalable font was just to have some kind of scalable font for start, from practical perspective we will need to be able to render normal fonts. And every char from this font is rendered into square, so that is why there is only one fixed size for width and height.

I like idea with font stored in PNG file, I did not thought about that before. I am surely for implementing it.

> I used the BleskOS calculator for this and it is not yet very efficient

Well, there are at least one thousand things that needs to be added to Calculator 😅 

However I will now take some break from programming BleskOS, maybe about one week. But I will merge PRs if you will add some improvements.
